lib/fork.js

```diff
@@ -14,8 +14,8 @@ module.exports = function (args) {
 	var file = args[0];
 
 	var options = {
-		silent: true,
-		cwd: path.dirname(file)
+		cwd: path.dirname(file),
+		stdio: ['ignore', process.stdout, process.stdout]
```

Comment:
Can you open an issue with the contents of this comment? I'm not sure to be honest.